src/main/resources/liquibase/crop_changelog/15_4_1.xml

```diff
@@ -0,0 +1,190 @@
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8196';
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8179'; 
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8200';
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8202';
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8204'; 
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8206';
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8208';
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8154'; 
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8210';
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8146';
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8080'; 
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8082';
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8078';
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8074'; 
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8076';
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8084';
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8212';
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8214';
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8216';
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8198'; 
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8148';
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8062'; 
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8068';
+
+		       SET FOREIGN_KEY_CHECKS=1;
+		</sql>
+	</changeSet>
+	
+	<changeSet id="v15.4.1-2" author="rowena">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="0">
+				SELECT COUNT(*) FROM cvterm WHERE cvterm_id = 6000;
+			</sqlCheck>
+		</preConditions>
+		<comment>ADD location ID 6000 AND SET AS DEFAULT</comment>
+		<sql dbms="mysql">
+			INSERT INTO location (`locid`, `ltype`, `nllp`, `lname`, `labbr`, `snl3id`, `snl2id`, `snl1id`, `cntryid`, `lrplce`, `nnpid`, `program_uuid`) VALUES
+			(6000,1500,0,'Default Seed Store','DSS',0,0,0,0,0,0,NULL),
+			(6001,410,0,'Default Breeding Location','DBL',0,0,0,0,0,0,NULL);
+		</sql>
+	</changeSet>
+
+	<changeSet id="v15.4.1-3" author="rowena">
+		<preConditions onFail="MARK_RAN">
+			<and>
+				<columnExists tableName="location" columnName="ldefault"/>
+				<sqlCheck expectedResult="0">
+					SELECT COUNT(1) FROM location l
+					WHERE l.ldefault = 1 AND l.ltype =
+					(SELECT u.fldno FROM udflds u WHERE u.ftable='LOCATION' AND u.ftype='LTYPE' AND fcode='SSTORE')
+				</sqlCheck>
+				<sqlCheck expectedResult="1">
+					SELECT COUNT(1) FROM location WHERE locid=6000 AND AND ltype = 1500;
+				</sqlCheck>
+			</and>
+		</preConditions>
+		<comment>SET DSS AS DEFAULT STORAGE location</comment>
+		<sql dbms="mysql">
+			UPDATE location SET ldefault = 1 WHERE locid=6000;
+		</sql>
+	</changeSet>
+	
+	<changeSet id="v15.4.1-4" author="rowena">
+		<preConditions onFail="CONTINUE">
+			<sqlCheck expectedResult="1">
+				SELECT COUNT(1) FROM udflds u WHERE  fcode = "ANCEST";
```

Comment:
better add these too to the precondition: `AND ftable = "ATRIBUTS" AND fldno>1551`

Reply:
@rvtulod I added `AND ftable = "ATRIBUTS" AND fldno>1551` to the precondition because 

> SELECT COUNT(1) FROM udflds u WHERE  fcode = "ANCEST";
retrieves 2 records in wheat DB.